babel/babel.c

```diff
@@ -165,7 +165,8 @@ static ssize_t dev_read(struct file *filep, char *buffer, size_t len, loff_t *of
  */
 static ssize_t dev_write(struct file *filep, const char *buffer, size_t len, loff_t *offset){
     //  Write the string into our message memory. Record the length.
-    copy_from_user(message, buffer, len);
+    long copied_from_user;
```

Comment:
Hi @mislam19 how does this variable then get used? I notice we don't use it in the comment or code afterwards is that deliberate?

Reply:
yes in short